packages/nextjs/src/config/webpack.ts

```diff
@@ -380,11 +380,33 @@ function checkWebpackPluginOverrides(
  * @returns `true` if sentry code should be injected, and `false` otherwise
  */
 function shouldAddSentryToEntryPoint(entryPointName: string, isServer: boolean): boolean {
-  return (
-    entryPointName === 'pages/_app' ||
-    (entryPointName.includes('pages/api') && !entryPointName.includes('_middleware')) ||
-    (isServer && entryPointName === 'pages/_error')
-  );
+  // On the server side, by default we inject the `Sentry.init()` code into every page (with a few exceptions).
```

Comment:
We already skip anything that doesn't start with `pages/`, so given that the `app/` folder is a sibling to, rather than child of, `pages/`, I think we're good on that score, no? Same for the middleware that lives alongside both.

Reply:
Ah yeah that makes sense. Didn't fully think it through. Thanks!